### DISCUSSION TITLE
#可以完善一下简体中文语言吗？

### DISCUSSION THREAD

**Discussion by faithysq666** [pawelsalawa/sqlitestudio#4309] [General]
#简体中文语言没有很好的支持，很多功能未能完全汉化

版本是3.3.3

**Comment by yfdyh000** [comment]
简体中文翻译已在最近改善。请尝试于 https://github.com/yfdyh000/sqlitestudio/actions?query=branch%3Aci-builds-now 构建的最新版本。欢迎提出意见。
